tests/components/sensor/test_dsmr.py

```diff
@@ -121,7 +121,7 @@ def test_derivative():
     }
     yield from entity.async_update()
 
-    assert entity.state == 1, \
+    assert entity.state == 3600, \
```

Comment:
Looking closer, this does not seem to test the time aspect at all?

Reply:
Glad you are being critical, I think quality is very important.
I think it does test the time aspect, because the first telegram is imported with timestamp '1' (and value 1) and the second telegram with timestamp '2' (and value 2).
That is if I'm reading the MBusObjects correctly.
Please correct me if I'm wrong.

Reply:
Ah, my bad ... I did not realize that `'value': 1` is in fact a timestamp (in seconds).

So, just to maybe understand this: is the issue that the data is in fact updated more frequently than the previous assumption of once per hour?

Reply:
Yes, according to the dsmr standard, the data is sent from the meter every 10 seconds for all versions < 5.
For version 5 (latest) the meter readings are updated every second.

Reply:
I see. Would you mind updating the test with realistic values so the next person to look at it does not get fooled like I did?

Reply:
Sure!
Will do so later tonight, as I am not home right now.

Reply:
I updated the test with values from my own smart meter.
My central heating was just in a state of low gas usage, so I had to wait a few seconds 😛.
In 330 seconds it used 0.003 m3 of gas, as you can see from the timestamps and m3 values.
The derived value came to the exact m3/h rate that I had calculated by hand before running the test (0,03272727272727272727272727272727) apart from some rounding.

Reply:
Awesome! I updated the test to ignore this issue, hope you don't mind.

Reply:
Awesome!